hack/00-osd-managed-cluster-config.selectorsyncset.yaml.tmpl

```diff
@@ -3304,6 +3304,165 @@ objects:
         apiGroup: rbac.authorization.k8s.io
         kind: Role
         name: dedicated-admins-openshift-customer-monitoring
+- apiVersion: hive.openshift.io/v1
+  kind: SelectorSyncSet
+  metadata:
+    labels:
+      managed.openshift.io/gitHash: ${IMAGE_TAG}
+      managed.openshift.io/gitRepoName: ${REPO_NAME}
+      managed.openshift.io/osd: 'true'
+    name: osd-devaccess
+  spec:
+    clusterDeploymentSelector:
+      matchLabels:
+        api.openshift.com/managed: 'true'
+    resourceApplyMode: Sync
+    resources:
+    - apiVersion: user.openshift.io/v1
+      kind: Group
+      metadata:
+        name: osd-devaccess
+      users: []
+    - apiVersion: rbac.authorization.k8s.io/v1
+      kind: ClusterRole
+      metadata:
+        name: osd-devaccess
+      rules:
+      - apiGroups:
+        - ''
+        resources:
+        - nodes
+        verbs:
+        - get
+        - list
+      - apiGroups:
+        - ''
+        resources:
+        - pods/exec
+        verbs:
+        - create
+      - apiGroups:
+        - config.openshift.io
+        resources:
+        - projects
+        verbs:
+        - get
+        - list
+      - apiGroups:
+        - project.openshift.io
+        resources:
+        - projects
+        verbs:
+        - get
+        - list
+      - apiGroups:
+        - user.openshift.io
+        resources:
+        - groups
+        verbs:
+        - get
+        - list
+      - apiGroups:
+        - ''
+        resources:
+        - namespaces
+        - namespaces/finalize
+        verbs:
+        - get
+        - list
+      - apiGroups:
+        - ''
+        resources:
+        - pods
+        verbs:
+        - get
+        - list
+      - apiGroups:
+        - security.openshift.io
+        resources:
+        - podsecuritypolicyreviews
+        - podsecuritypolicyselfsubjectreviews
+        - podsecuritypolicysubjectreviews
+        verbs:
+        - create
+      - apiGroups:
+        - authorization.openshift.io
+        resources:
+        - localresourceaccessreviews
+        verbs:
+        - create
+      - nonResourceURLs:
+        - '*'
+        verbs:
+        - '*'
+    - apiVersion: rbac.authorization.k8s.io/v1
+      kind: ClusterRoleBinding
+      metadata:
+        name: osd-devaccess
+      roleRef:
+        apiGroup: rbac.authorization.k8s.io
+        kind: ClusterRole
+        name: osd-devaccess
+      subjects:
+      - kind: Group
+        name: osd-devaccess
+    - apiVersion: v1
+      data:
+        ca.crt: '-----BEGIN CERTIFICATE-----
+
+          MIIENDCCAxygAwIBAgIJANunI0D662cnMA0GCSqGSIb3DQEBCwUAMIGlMQswCQYD
+
```

Comment:
This is formatted oddly? I wonder why it's like this and not the `|-` style as above?